spec/sql_footprint_spec.rb

```diff
@@ -28,14 +28,26 @@
       )
     end
 
-    it 'formats selects' do
+    it 'appends the statement count by 1' do
+      expect do
+        Widget.where(name: SecureRandom.uuid, quantity: 1).last
+      end.to change { statements.count }.by(+1)
+    end
+
+    it 'appends 1 sql statement' do
+      matches = []
       Widget.where(name: SecureRandom.uuid, quantity: 1).last
-      expect(statements.to_a).to include(
-        'SELECT  "widgets".* FROM "widgets" ' \
-        'WHERE "widgets"."name" = ? AND ' \
-        '"widgets"."quantity" = ?  ' \
-        'ORDER BY "widgets"."id" DESC LIMIT 1'
-      )
+
+      matches.push 'SELECT  "widgets".* FROM "widgets"'
+      matches.push 'WHERE "widgets"."name" = ?'
+      matches.push 'AND'
+      matches.push 'widgets"."quantity" = ?'
+      matches.push 'ORDER BY "widgets"."id" DESC'
+      matches.push 'LIMIT ?'
+
+      matches.each do |match|
+        expect(statements.to_a.detect { |record| record.match match }).not_to be_nil
```

Comment:
I stared at this for a while trying to think of a simpler way of stating it, but I've got nothing. Would be nice to simplify, if possible.